Makefile

```diff
@@ -43,6 +44,7 @@ NODE_EXE = node$(EXEEXT)
 NODE ?= ./$(NODE_EXE)
 NODE_G_EXE = node_g$(EXEEXT)
 NPM ?= ./deps/npm/bin/npm-cli.js
+TEST_CI = ./tools/test-ci.sh
```

Comment:
I'm pretty sure `--mode=release` is the default, so if we made `-J` the default as well then we definitely wouldn't need this indirection.

>But what about `vcbuild.bat`

This doesn't help with that though, you still have the `tools/test.py` command duplicated in Windows and Unix files.

>and the case of  #12830 & #12771

I think people should run `tools/test.py` directly, this discourages them from doing that.